phpseclib/Crypt/Random.php

```diff
@@ -148,13 +148,13 @@ function crypt_random_string($length)
             session_start();
 
             $v = $seed = $_SESSION['seed'] = pack('H*', sha1(
-                serialize($_SERVER) .
-                serialize($_POST) .
-                serialize($_GET) .
-                serialize($_COOKIE) .
-                serialize($GLOBALS) .
-                serialize($_SESSION) .
-                serialize($_OLD_SESSION)
+                phpseclib_safe_serialize($_SERVER) .
+                phpseclib_safe_serialize($_POST) .
+                phpseclib_safe_serialize($_GET) .
+                phpseclib_safe_serialize($_COOKIE) .
+                phpseclib_safe_serialize($GLOBALS) .
+                phpseclib_safe_serialize($_SESSION) .
+                phpseclib_safe_serialize($_OLD_SESSION)
```

Comment:
The following:
```php
echo phpseclib_safe_serialize($_SESSION);
echo phpseclib_safe_serialize($_OLD_SESSION);
```

gives the following notices on PHP versions 4.4.9 to 7.0.3 on http://sandbox.onlinephpfunctions.com/ (possibly due to PHP.INI configuration?):
```
<br />
<b>Notice</b>:  Undefined variable: _SESSION in <b>[...][...]</b> on line <b>32</b><br />
N;---;<br />
<b>Notice</b>:  Undefined variable: _OLD_SESSION in <b>[...][...]</b> on line <b>33</b><br />
```

Maybe an existence check using `isset()` is needed before using the variables with `phpseclib_safe_serialize()`?